### PR TITLE
Make truncation an option, fix runtime error from updated pytorch

### PR DESCRIPTION
**Pull request by alexholdenmiller**
fix pytorch bug reported in #270
default to truncating long inputs, but can turn this off with -tr false

### CODE REVIEW

**Comment by alexholdenmiller** [comment]
(is truncating on a good default? or should we default off? truncation values currently need to be modified manually)

File: parlai/agents/seq2seq/seq2seq.py [hunk 1]
```diff
@@ -49,6 +49,9 @@ def add_cmdline_args(argparser):
             help='rank candidates if available. this is done by computing the' +
                  ' mean score per token for each candidate and selecting the ' +
                  'highest scoring one.')
+        agent.add_argument('-tr', '--truncate', type='bool', default=True,
```
Comment (jaseweston) [inline parlai/agents/seq2seq/seq2seq.py]: explain what this means in the comment please

Comment (jaseweston) [inline parlai/agents/seq2seq/seq2seq.py]: like...remove padding or?

**Comment by alexholdenmiller** [comment]
this look okay now, @jaseweston?

**Pull request merged**